admin/app/components/solidus_admin/users/edit/component.html.erb

```diff
@@ -0,0 +1,62 @@
+<%= page do %>
+  <%= page_header do %>
+    <%= page_header_back(solidus_admin.users_path) %>
+    <%= page_header_title(t(".title", email: @user.email)) %>
+
+    <% # @todo: I am not sure how we want to handle Cancan stuff in the new admin. %>
+    <% # if can?(:admin, Spree::Order) && can?(:create, Spree::Order) %>
+      <%= page_header_actions do %>
+        <%= render component("ui/button").new(tag: :a, text: t(".create_order_for_user"), href: spree.new_admin_order_path(user_id: @user.id)) %>
+      <% end %>
+    <% # end %>
+  <% end %>
+
+  <%= page_header do %>
+    <% tabs.each do |tab| %>
+      <%= render(component("ui/button").new(tag: :a, scheme: :ghost, text: tab[:text], 'aria-current': tab[:current], href: tab[:href])) %>
+    <% end %>
+  <% end %>
+
+  <%= page_with_sidebar do %>
+    <%= page_with_sidebar_main do %>
+
+      <%= render component('ui/panel').new(title: Spree.user_class.model_name.human) do %>
+        <%= form_for @user, url: solidus_admin.user_path(@user), html: { id: form_id } do |f| %>
+          <div class="py-1.5">
+          <%= render component("ui/forms/field").text_field(f, :email) %>
```

Comment:
Could we add `autocomplete: none` so that (as visible in your video) password managers and browser auto complete does not kick in? I think it does not make any sense in an admin interface.

_*) indentation of the form fields seems odd_

Reply:
I can definitely add that! But I think the autocomplete that you can see in the video is actually part of the old admin page (the add new user flow shown at the start of the video). This PR only adds the ability to view and edit users that already exist. Happy to add regardless if you still think that's an issue though!

Reply:
I think so, yes. Would you mind adding it? Also please fix the indentation.

Reply:
Fixed the indentation and added `html: { id: form_id, autocomplete: "off" }` to the whole form rather than the individual fields as I don't think autocomplete makes sense at all here. Sorry for the delay, was on holiday last week!

Reply:
No worries!